thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -34,6 +33,8 @@ export default Ember.Controller.extend({
   //
   // services
   //
+  authService: Ember.inject.service('session'),
```

Comment:
any reason why we changed the name here? (curious)

Reply:
Naming collision. 'session' currently refers to 'RCA session' in the route and controller - this has to be refactored in the future

Reply:
fixed